setup.cfg

```diff
@@ -1,3 +1,43 @@
 [flake8]
 max-line-length = 88
 extend-ignore = E203
+exclude = build, ci-helpers
+ignore =
```

Comment:
this seems to have caused a bunch of new inline errors, probably worth reverting

Reply:
Apparently the "ignore" should not be overriden, and the exclude is no longer needed so I'll revert that.